scripts/build-release.sh

```diff
@@ -5,10 +5,18 @@
 
 set -e
 
```

Comment:
This is more a general issue that we build targets independently, but we require the full cmake configuration step to do so.

Might make sense to go further down this path and add `LD_BUILD_SERVER` and `LD_BUILD_CLIENT` so we can speed up CI / disable unecessary work for people who don't want it. Or, we could have different cmake projects and pull them together into a superbuild.